cmake/kokkos_tribits.cmake

```
@@ -488,7 +488,7 @@ FUNCTION(KOKKOS_LIB_COMPILE_DEFINITIONS)
   IF(KOKKOS_HAS_TRILINOS)
     #don't trust tribits to do this correctly
     KOKKOS_TARGET_COMPILE_DEFINITIONS(${TARGET} ${ARGN})
-  ELSE(TARGET ${TARGET})
+  ELSEIF(TARGET ${TARGET})
```

@jjwilke did you mean this or
```suggestion
  ELSE()
```
?

---

Yeah that would make more sense to me too.